pkg/controller/template/test_data/templates/worker/01-worker-kubelet/vpshere/files/-etc-kubernetes-kubelet.conf

```diff
@@ -1,6 +1,6 @@
 contents:
-  source: data:,kind%3A%20KubeletConfiguration%0AapiVersion%3A%20kubelet.config.k8s.io%2Fv1beta1%0AcgroupDriver%3A%20systemd%0AclusterDNS%3A%0A%20%20-%2010.3.0.10%0AclusterDomain%3A%20cluster.local%0AmaxPods%3A%20250%0ArotateCertificates%3A%20true%0AruntimeRequestTimeout%3A%2010m%0AserializeImagePulls%3A%20false%0AstaticPodPath%3A%20%2Fetc%2Fkubernetes%2Fmanifests%0AsystemReserved%3A%0A%20%20cpu%3A%20500m%0A%20%20memory%3A%20500Mi%0AfeatureGates%3A%0A%20%20RotateKubeletServerCertificate%3A%20true%0AserverTLSBootstrap%3A%20true%0A
```

Comment:
uhm, don't we need these anymore?

Reply:
not sure why that happened.

OH that whole folder seems to be misnamed?? `vpshere`??

Reply:
fix here:
https://github.com/openshift/machine-config-operator/pull/589/